app/abilities/lecture_ability.rb

```diff
@@ -26,7 +26,7 @@ def initialize(user)
 
     can :search, Lecture
 
-    can [:show_announcements, :organizational, :show_structures,
+    can [:show_announcements, :organizational,
          :search_examples, :show_random_quizzes,
```

Comment:
`search_examples` should be removed as well, right?

Reply:
Furthermore, does `structure_path` in `pdf_uploader.rb` refer to something Erdbeere-related?

Reply:
Thanks for noticing. Yes, `search_examples` should be removed as well. I fixed that. The `structure_path` in the uploader file is unrelated to Erdbeere.